pallet-chainlink-feed/src/benchmarking.rs

```diff
@@ -0,0 +1,744 @@
+use super::*;
+use frame_benchmarking::{account, benchmarks, whitelisted_caller};
+use frame_support::traits::Get;
+use frame_system::RawOrigin;
+use sp_std::fmt::Debug;
+
+use crate::Module as ChainlinkFeed;
+
+const SEED: u32 = 0;
+
+/// Either use `assert_ok!` or regular `assert!` depending on std/no_std
+/// environment.
+fn assert_is_ok<T: Debug, E: Debug>(r: Result<T, E>) {
+	#[cfg(feature = "std")]
+	frame_support::assert_ok!(r);
+	#[cfg(not(feature = "std"))]
+	assert!(r.is_ok());
+}
+
+fn whitelisted_account<T: Trait>(name: &'static str, counter: u32) -> T::AccountId {
+	let acc = account(name, counter, SEED);
+	whitelist_acc::<T>(&acc);
+	acc
+}
+
+fn whitelist_acc<T: Trait>(acc: &T::AccountId) {
+	frame_benchmarking::benchmarking::add_to_whitelist(
+		frame_system::Account::<T>::hashed_key_for(acc).into(),
+	);
+}
+
+benchmarks! {
+	_ {}
```

Comment:
this is deprecated in some later version of substrate fyi

Reply:
yep, but it's necessary for v2.0.1, right?